beetsplug/replaygain.py

```diff
@@ -1402,5 +1480,24 @@ def func(lib, opts, args):
         cmd.parser.add_option(
             "-W", "--nowrite", dest="write", action="store_false",
             help=u"don't write metadata (opposite of -w)")
-        cmd.func = func
+        cmd.func = self.replaygain_func
         return [cmd]
+
+    def replaygain_func(self, lib, opts, args):
+        """Handle `replaygain` ui subcommand
+        """
+        write = ui.should_write(opts.write)
+        force = opts.force
+
+        if opts.threads != 0:
```

Comment:
It seems like this condition might be supposed to be `!= 1`? Or better yet, the condition should come after checking the config file too?

Reply:
* I wanted to include the option to completely bypass `ThreadPool` from the CLI with `--threads 0`, seems like a useful feature down the line in case we ever suspect the parallel processing is messing something up
* `ThreadPool(1)` still provides one worker thread that can work asynchronously which may be useful in some cases? I don’t know.